telemetry/definitions/commonDefinitions.json

```diff
@@ -855,6 +855,51 @@
             "name": "userChoice",
             "type": "string",
             "description": "User selection from a predefined menu (not user-provided input)" 
+        },
+        {
+            "name": "codewhispererterminal_command",
+            "type": "string",
+            "description": "The CLI tool a completion was for"
+        },
+        {
+            "name": "codewhispererterminal_duration",
+            "type": "int",
+            "description": "A duration in milliseconds"
+        },
+        {
+            "name": "codewhispererterminal_duration",
+            "type": "int",
+            "description": "A duration in milliseconds"
+        },
+        {
+            "name": "codewhispererterminal_timeToSuggestion",
+            "type": "int",
+            "description": "How long the AI model roundtrip took"
+        },
+        {
+            "name": "codewhispererterminal_accepted",
+            "type": "boolean",
+            "description": "Whether the user accepted a completion"
+        },
+        {
+            "name": "codewhispererterminal_typedCount",
+            "type": "int",
+            "description": "A count of characters typed"
+        },
+        {
+            "name": "codewhispererterminal_suggestedCount",
+            "type": "int",
+            "description": "A count of characters accepted"
+        },
+        {
+            "name": "codewhispererterminal_itemName",
+            "type": "string",
```

Comment:
just 'item' is too abstract in description. it would be helpful to know what do we consider as item.